sunpy/data/test/__init__.py

```diff
@@ -41,4 +45,26 @@ def get_test_filepath(filename, **kwargs):
     return get_pkg_data_filename(filename, package="sunpy.data.test", **kwargs)
 
 
-file_list = glob.glob(os.path.join(rootdir, '*.[!p]*'))
+def test_data_filenames():
+    """
+    Return a list of all test files in ``data/test`` directory.
+
+    This ignores any py, pyc and __*__ files in these directories.
```

Comment:
these suffixes and especially the underscores need to be in double backticks 
```
``__*__``
```

to prevent them being  interpreted as rst markup.

Reply:
Thats my bad.